<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1129</width>
    <height>715</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>변환기</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="1" rowspan="3">
         <widget class="QScrollArea" name="scrollArea">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>98</width>
             <height>605</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QPushButton" name="btn_util_hex">
              <property name="text">
               <string>hex</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btn_util_decimal">
              <property name="text">
               <string>decimal</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btn_util_binuary">
              <property name="text">
               <string>biunary</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btn_util_base64">
              <property name="text">
               <string>base64</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btn_util_ascii">
              <property name="text">
               <string>ascii (문자)</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="Line" name="line">
              <property name="orientation">
               <enum>Qt::Orientation::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btn_short_big">
              <property name="text">
               <string>Short B (2)</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btn_double_big">
              <property name="text">
               <string>Double B (8)</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btn_int_big">
              <property name="text">
               <string>int B(4)</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btn_float_big">
              <property name="text">
               <string>Float B(4)</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="Line" name="line_2">
              <property name="orientation">
               <enum>Qt::Orientation::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btn_short_little">
              <property name="text">
               <string>Short L (2)</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btn_double_little">
              <property name="text">
               <string>Double L (8)</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btn_int_little">
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="text">
               <string>int L(4)</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btn_float_little">
              <property name="text">
               <string>Float L(4)</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QComboBox" name="util_combo"/>
        </item>
        <item row="1" column="0" rowspan="2">
         <widget class="QTextEdit" name="util_text"/>
        </item>
        <item row="0" column="2" rowspan="3">
         <widget class="QTabWidget" name="tabWidget_2">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="text">
           <attribute name="title">
            <string>text</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="0" column="0">
             <widget class="QTextEdit" name="util_encode"/>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_3">
           <attribute name="title">
            <string>table</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_4">
            <item row="0" column="0">
             <widget class="QTableWidget" name="util_encode_table"/>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1129</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
